cli/dcoscli/metrics.py

```diff
@@ -9,6 +9,11 @@
 emitter = emitting.FlatEmitter()
 
 
+class EmptyMetricsException(DCOSException):
+    def __init__(self):
+        DCOSException.__init__(self, 'No metrics found')
```

Comment:
As we don't have to support Python 2, could you switch this to `super().__init__( 'No metrics found')`?

Reply:
👍